spark/src/test/java/org/apache/iceberg/spark/source/TestSparkTableUtil.java

```diff
@@ -162,4 +163,33 @@ public void testImportUnpartitionedTable() throws Exception {
     long count = spark.read().format("iceberg").load(location.toString()).count();
     Assert.assertEquals("three values ", 3, count);
   }
+
+  @Test
+  public void testImportAsHiveTable() throws Exception {
+    spark.table(qualifiedTableName).write().mode("overwrite").format("parquet")
+            .saveAsTable("unpartitioned_table");
+    TableIdentifier source = spark.sessionState().sqlParser()
```

Comment:
What about simply constructing `TableIdentifier`? For example, `new TableIdentifier("unpartitioned_table")`?

Reply:
It works. Agreed to use the simpler one.